lib/ftpoison/base.ex

```diff
@@ -88,9 +88,9 @@ defmodule FTPoison.Base do
         :inets.start
       end
 
-      @spec to_char_list(String.t) :: List.t
-      defp to_char_list(string) do
-        string |> String.to_char_list
+      @spec to_charlist(String.t) :: List.t
+      defp to_charlist(string) do
+        string |> String.to_charlist
```

Comment:
This was a 1.5 deprecation.
https://github.com/elixir-lang/elixir/blob/v1.5/CHANGELOG.md